src/main.ts

```diff
@@ -4,13 +4,14 @@ import * as exec from '@actions/exec';
 import * as io from '@actions/io';
 
 import { FormatType, SecretParser } from 'actions-secret-parser';
+import { ServicePrincipalLogin } from './PowerShell/ServicePrincipalLogin';
 
 var azPath: string;
 var prefix = !!process.env.AZURE_HTTP_USER_AGENT ? `${process.env.AZURE_HTTP_USER_AGENT}` : "";

```

Comment:
Can you check if the same variable is used as user-agent in powershell. I think the powershell team send some data in an email as for how to add data for telemetry in their user-agent.